### PR TITLE
Add Fobnail Github main page

### PR DESCRIPTION
**Pull request by Pokisiekk**
Signed-off-by: Kamil Pokornicki <kamil.pokornicki@3mdeb.com>

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+          # - id: clippy
+          #   name: 'clippy: pal_pc'
+          #   args: ['--manifest-path', 'pal/pal_pc/Cargo.toml', '--', '-D', 'warnings']
+
+  - repo: https://github.com/crate-ci/typos
+    rev: v1.10.2
+    hooks:
+      - id: typos
+
+# cargo is not supported for now in the pre-commit.ci
```
Comment (krystian-hebel) [inline .pre-commit-config.yaml]: There is no Rust in this repo, all of these can be removed.

File: profile/README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
+## Hi there, we are Fobnail Team 👋
+
+The Fobnail project provides the resources to create an axiomatically
+trustworthy device and a simple user interface to confirm the status of the
+platform. We aim to provide a reference architecture for building
+offline integrity measurement verifiers on a USB device and validators running
+in dynamically run measured environments (DLME).
+
+Our goal is to create a scalable, flexible implementation that will be the main
+building block of future (community) security solutions based on DRTM payloads.
```
Comment (krystian-hebel) [inline profile/README.md]: We are not using DRTM payloads and won't be in the foreseeable future. Whoever proposed it didn't know about the amount of work that is required to add support for DRTM payloads to an OS.

File: profile/README.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+The owner of Fobnail can verify the reliability of the working system before
+performing any sensitive operation. Fobnail does not need an internet
+connection, which makes it immune to network stack attacks and remote
+infrastructure.
```
Comment (krystian-hebel) [inline profile/README.md]: Platform provisioning requires internet connection to obtain TPM certificates' chains as of now. Fobnail Token presents itself as a network device, so it uses network stack.

I don't get last part of this sentence. Fobnail is immune to remote infrastructure?

File: profile/README.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+[The Fobnail Token](https://fobnail.3mdeb.com/fobnail_token/) is a small
+open-source USB hardware reference project that allows a
+user/administrator/company to determine system integrity.
```
Comment (krystian-hebel) [inline profile/README.md]: In my opinion this sentence suggests that we designed this dongle as part of Fobnail, which is not true. We are using an existing hardware (nRF52840 Dongle) for a reference implementation of Fobnail Token.

File: profile/README.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+The Fobnail project has the potential to scale up in the future. The Fobnail
+architecture can be adapted to devices using other buses than USB.
+A detailed description of the project can be found
+[here](https://fobnail.3mdeb.com/).
```
Comment (krystian-hebel) [inline profile/README.md]: The last sentence doesn't fit with the rest, it should be in separate paragraph, maybe together with the next one.

File: profile/README.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+Fobnail is a product developed by [3mdeb](https://github.com/3mdeb).
```
Comment (krystian-hebel) [inline profile/README.md]:
```suggestion
Fobnail is developed by [3mdeb](https://github.com/3mdeb).
```

I don't think we can call it a product yet.

File: profile/README.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+
+## Community importance
+
+We believe that a friendly community and cooperation are key to building
+open-source trustworthy solutions. Because of that, we would like to invite you
+to:
+
+* [Fobnail Matrix Community](https://matrix.to/#/#dasharo:matrix.org) - where we
```
Comment (krystian-hebel) [inline profile/README.md]:
```suggestion
* [Fobnail Matrix Community](https://matrix.to/#/#fobnail:matrix.org) - where we
```

File: profile/README.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+
+We also maintain our forks and open-source projects like:
+
+* [cortex-mpu](https://github.com/fobnail/cortex-mpu)
+* [littlefs2-sys](https://github.com/fobnail/littlefs2-sys)
+* [littlefs2](https://github.com/fobnail/littlefs2)
+* [nrf-hal](https://github.com/fobnail/nrf-hal)
+* [lru_time_cache](https://github.com/fobnail/lru_time_cache)
+* [coap-lite](https://github.com/fobnail/coap-lite)
+* [coap-server-rs](https://github.com/fobnail/coap-server-rs)
```
Comment (krystian-hebel) [inline profile/README.md:49]: @arturkow2000 are all of this still used?

**Review by krystian-hebel — CHANGES_REQUESTED** [review]

File: .pre-commit-config.yaml (hunk 1 above)
Comment (Pokisiekk) [reply .pre-commit-config.yaml]: Removed.

File: profile/README.md (hunk 2 above)
Comment (Pokisiekk) [reply profile/README.md]: Part with DRTM was removed.
Most of the information I took from [this documentation](https://fobnail.3mdeb.com/), which I see is heavily outdated.

File: profile/README.md (hunk 3 above)
Comment (Pokisiekk) [reply profile/README.md]: Sentence was removed, also took information from fobnail documentation...

File: profile/README.md (hunk 4 above)
Comment (Pokisiekk) [reply profile/README.md]: Updated

File: profile/README.md (hunk 5 above)
Comment (Pokisiekk) [reply profile/README.md]: Separated and connected with next sentence.

File: profile/README.md (hunk 8 above)
Comment (arturkow2000) [reply profile/README.md:49]: @krystian-hebel Yes, all of them are still used

**Review by krystian-hebel — APPROVED** [review]

**Pull request merged**